revealer_bot/decryption_action.py

```diff
@@ -14,9 +14,7 @@ async def decrypt_attached_tmk(message):
     if message.attachments:
         url = message.attachments[0].url
         tmk_filename = message.attachments[0].filename
-        await message.reply(
```

Comment:
Just formatting changes stemming from increasing max-line-length to 160 characters